### PR TITLE
Align LoadingMediaDisplay positino to TextMediaDisplay

### PR DESCRIPTION
**Pull request by laiyichin**
#### WHAT
Align LoadingMediaDisplay position to TextMediaDisplay and MarqueeMediaDisplay text position

#### Checklist :clipboard:
- [N/A] Add explicit visibility modifier and explicit return types for public declarations
- [X] Run spotless check
- [X] Run tests
- [X] Update metalava's signature text files
text_alignment

### CODE REVIEW

File: media/ui/src/main/java/com/google/android/horologist/media/ui/components/display/LoadingMediaDisplay.kt [hunk 1]
```diff
@@ -33,28 +32,30 @@ import androidx.wear.compose.material.MaterialTheme
 import com.google.android.horologist.annotations.ExperimentalHorologistApi
 
 /**
- * A loading state display.
+ * A loading state display. This style is matched to the TextViews of the and [TextMediaDisplay]
```
Comment (yschimke) [inline media/ui/src/main/java/com/google/android/horologist/media/ui/components/display/LoadingMediaDisplay.kt:35]: Should we mention MarqueeTextMediaDisplay as well?

Comment (yschimke) [inline media/ui/src/main/java/com/google/android/horologist/media/ui/components/display/LoadingMediaDisplay.kt:35]:
```suggestion
 * A loading state display. This style is matched to the Text of [TextMediaDisplay] as [MarqueeTextMediaDisplay]
```

Comment (yschimke) [inline media/ui/src/main/java/com/google/android/horologist/media/ui/components/display/LoadingMediaDisplay.kt:35]: If we actually care about this, we should probably consider how we prove this.

Either 

1) The components use the same layout structure, so it's clear they match.  But I don't think this is easy.

2) Maybe a player screen shot with both layouts, with placeholders on top at 50% alpha?

Can be for later.

**Review by yschimke — APPROVED** [review]

File: media/ui/src/main/java/com/google/android/horologist/media/ui/components/display/LoadingMediaDisplay.kt (hunk 1 above)
Comment (laiyichin) [reply media/ui/src/main/java/com/google/android/horologist/media/ui/components/display/LoadingMediaDisplay.kt]: Great suggestion! Added screenshots overlaying them. Thank you!

**Pull request merged**